### PR TITLE
QobjEvo Bench

### PR DESCRIPTION
**Pull request by xspronken**
Benchmark file for QobjEvo.

### CODE REVIEW

File: benchmarks/tests/test_lin_alg_QobjEvo.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        def cos_t(t):
+            return np.cos(t)
+        return qutip.QobjEvo([res, cos_t])
+
+    if coeftype == 'string':
+        return qutip.QobjEvo([res, 'cos(t)'])
+
+    tlist = np.linspace(0, 10, 101)
+    values = np.cos(tlist)
+    return qutip.QobjEvo([res, values], tlist=tlist)
```
Comment (Ericgig) [inline benchmarks/tests/test_lin_alg_QobjEvo.py:48]: Could you rewrite this to have only one `return`. Or at lleast a block for `array`.

Comment (xspronken) [reply benchmarks/tests/test_lin_alg_QobjEvo.py]: Fixed in [43f1491](https://github.com/qutip/qutip-benchmark/pull/7/commits/43f14910f17cd4c2fda5880ebdc25759207e0c6e). 
Should I use the same tlist for all coefficients or leave it as None for string and function ?

Comment (Ericgig) [reply benchmarks/tests/test_lin_alg_QobjEvo.py]: `tlist` is ignored for other types so you can leave it like this..

**Review by Ericgig — APPROVED** [review]
Look good.

**Comment by xspronken** [comment]
@hodgestar if you approve i can merge the PR.

**Review by hodgestar — APPROVED** [review]
Looks good.

**Pull request merged**